tests/Jellyfin.Naming.Tests/Video/VideoResolverTests.cs

```diff
@@ -1,275 +1,205 @@
-                parser.ResolveFile(@"/server/Movies/brave (2007)/brave (2006).3d.sbs.mkv");
-
-            Assert.Equal(2006, result.Year);
-            Assert.False(result.IsStub);
-            Assert.True(result.Is3D);
-            Assert.Equal("sbs", result.Format3D);
-            Assert.Equal("brave", result.Name);
-            Assert.Null(result.ExtraType);
-        }
-
-        [Fact]
-        public void TestNameWithoutDate()
-        {
-            var parser = GetParser();
-
-            var result =
-                parser.ResolveFile(@"/server/Movies/American Psycho/American.Psycho.mkv");
-
-            Assert.Null(result.Year);
-            Assert.False(result.IsStub);
-            Assert.False(result.Is3D);
-            Assert.Null(result.Format3D);
-            Assert.Equal("American.Psycho", result.Name);
-            Assert.Null(result.ExtraType);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateAndStringsSequence()
-        {
-            var parser = GetParser();
-
-            // In this test case, running CleanDateTime first produces no date, so it will attempt to run CleanString first and then CleanDateTime again
-            var result =
-                parser.ResolveFile(@"/server/Movies/3.Days.to.Kill/3.Days.to.Kill.2014.720p.BluRay.x264.YIFY.mkv");
-
-            Assert.Equal(2014, result.Year);
-            Assert.False(result.IsStub);
-            Assert.False(result.Is3D);
-            Assert.Null(result.Format3D);
-            Assert.Equal("3.Days.to.Kill", result.Name);
-            Assert.Null(result.ExtraType);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateAndStringsSequence1()
-        {
-            var parser = GetParser();
-
-            // In this test case, running CleanDateTime first produces no date, so it will attempt to run CleanString first and then CleanDateTime again
-            var result =
-                parser.ResolveFile(@"/server/Movies/3 days to kill (2005)/3 days to kill (2005).mkv");
-
-            Assert.Equal(2005, result.Year);
-            Assert.False(result.IsStub);
-            Assert.False(result.Is3D);
-            Assert.Null(result.Format3D);
-            Assert.Equal("3 days to kill", result.Name);
-            Assert.Null(result.ExtraType);
-        }
-
-        [Fact]
-        public void TestFolderNameWithExtension()
-        {
-            var parser = GetParser();
-
-            var result =
-                parser.ResolveFile(@"/server/Movies/7 Psychos.mkv/7 Psychos.mkv");
-
-            Assert.Null(result.Year);
-            Assert.False(result.IsStub);
-            Assert.False(result.Is3D);
-            Assert.Equal("7 Psychos", result.Name);
-            Assert.Null(result.ExtraType);
+        private readonly NamingOptions _namingOptions = new NamingOptions();
+
+        private class ResolveFileTestData : IEnumerable<object[]>
+        {
+            public IEnumerator<object[]> GetEnumerator()
```

Comment:
You could make this more concise with an internal method returning `IEnumerable<VideoFileInfo>` and then this would become

```cs
GetResolveFileTestDataInternal()
    .Select(d => new object[] { d })
    .GetEnumerator()
```